src/components/SentryTestPressable.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement, useState } from 'react'
+import { Pressable } from 'react-native'
+
+import { reportError } from '../services/sentry'
+
+interface Props {
+  children: ReactElement
+}
+
+const SentryTestPressable = ({ children }: Props): ReactElement => {
+  const [counter, setCounter] = useState<number>(0)
+  const CLICKS_TO_ENABLE_SENTRY = 20
+
+  const onPress = () => {
+    if (counter > CLICKS_TO_ENABLE_SENTRY) {
+      reportError('Error for testing Sentry')
```

Comment:
Perhaps it makes sense to reset it to zero after sending one report. Otherwise if you don't count the clicks you'll send multiple reports.

Reply:
@SteffiStuffel 

Oh sorry, I did oversee this. I would suggest to throw here a native (Android/Swift) or JS exception instead. Else it is not possible to test the whole stack.

So the app should really crash in this example. If the app actually stops, then crash reports get sent with the next start of the app. This should also be testable.

I'm not sure how we do it in Integreat, but somehow we are really crashing the Integreat app.

Reply:
Something like this: https://github.com/digitalfabrik/integreat-app/blob/f21b485038305188d7041f03f380ce3a592f6e43/native/src/utils/createSettingsSections.ts#L149

Reply:
Added it.